### DISCUSSION TITLE
Delete this post plz

### DISCUSSION THREAD

**Discussion by ghost** [skyline-emu/skyline#383] [General]
I want to use skyline on galaxy a12 mid range device yet i cant because you stupid monks only care about making money through sales of higher end hardware
Vulkan is a form of planned obsolescence that artificialy limits capabliyies of good smausng a12 mid range devices to force people to buy xiaomi crap

**Comment by bylaws** [comment]
This continues to make my day, I'm especially appreciative of the fact that you refer to us as monks. May I ask you though if we are monks then who is our God?

**Comment by ghost** [comment]
Ok give me a version of skyline that uses opengl2 instead of vulkan
I dont know how to write code so do it for me

**Comment by bylaws** [comment]
Who is our God?

**Comment by ghost** [comment]
I dont know just write a version with no vulkan and open gl 2 i have mid range helio p35 mtk

**Reply by jduncanator** [reply]
Just to let you know, I have completed just writing a version with no vulkan and open gl 2. It is working nicely. I will give it to you for $2,000.

**Reply by ghost** [reply]
I know you are joking but i dont have any money to pay

**Comment by ghost** [comment]
Can you give for free i got no money to pay and even then it would be cheaper to just buy vulkan supported device but i dont have money to buy anything

**Comment by ghost** [comment]
I know you are joking and you actually dont have it

**Comment by jduncanator** [comment]
I know you are joking and you actually dont hate it

**Comment by QuackingCanary** [comment]
Please grow up ... That's not a school playground

**Comment by AndresNorm** [comment]
And i guess he doesn't have a switch, to dump the game.

What a nuisance.
This is not an issue, though.
